Sources/App/Features/Sponsors/Transformers/SponsorTransformer.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SponsorTransformer.swift
+//  
+//
+//  Created by Alex Logan on 01/08/2022.
+//
+
+import Foundation
+
+enum SponsorTransformer: Transformer {
+    static func transform(_ entity: Sponsor?) -> SponsorResponse? {
+        guard let entity = entity else { return nil }
+        return .init(
+            id: entity.id,
+            name: entity.name,
+            image: "https://swiftleeds-speakers.s3.eu-west-2.amazonaws.com/\(entity.image)", // Hard coded in absence of environment variables
```

Comment:
I think we can use the env variables in this now as the key is live on heroku being used in the web sponsor code.